source/development/packaging/generate-dashboard-package.rst

```diff
@@ -144,119 +148,120 @@ Using the script
-      # cd wazuh-dashboard/dev-tools/build-packages/
+   - A system with Docker.
+   - Internet connection (to download the docker images the first time).
 
-#. Build the Docker image with the following parameters:
+.. topic:: Steps
 
-   -  ``NODE_VERSION``: Node version to use in the ``.nvmrc`` file.
-   -  ``WAZUH_DASHBOARDS_BRANCH``: Branch of the Wazuh dashboards repository.
-   -  ``WAZUH_DASHBOARDS_PLUGINS``: Branch of the Wazuh dashboards Plugins repository.
-   -  ``WAZUH_SECURITY_DASHBOARDS_PLUGIN_BRANCH``: Branch of the Wazuh Security Dashboards Plugin repository.
-   -  ``OPENSEARCH_DASHBOARDS_VERSION``: Version of the OpenSearch Dashboards. You can find the version in the ``package.json`` file of the Wazuh dashboards repository.
-   -  ``-t``: Tag of the image.
+   #. Clone the Wazuh dashboard repository and move to the ``dev-tools/build-packages/base-packages-to-base`` folder of the repository.
 
-   .. code:: console
-   
-      # docker build \
-      --build-arg NODE_VERSION=<NODE_VERSION> \
-      --build-arg WAZUH_DASHBOARDS_BRANCH=<BRANCH_OF_wazuh-dashboard> \
-      --build-arg WAZUH_DASHBOARDS_PLUGINS=<BRANCH_OF_wazuh-dashboard-plugins> \
-      --build-arg WAZUH_SECURITY_DASHBOARDS_PLUGIN_BRANCH=<BRANCH_OF_wazuh-security-dashboards-plugin> \
-      --build-arg OPENSEARCH_DASHBOARDS_VERSION=<OPENSEARCH_DASHBOARDS_VERSION> \
-      -t <TAG_OF_IMAGE> \ 
-      -f wazuh-dashboard.Dockerfile .
+      .. code:: console
+      
+         # git clone -b <BRANCH_OR_TAG> https://github.com/wazuh/wazuh-dashboard.git
+         # cd wazuh-dashboard/dev-tools/build-packages/
+      
+      Example:
+      
+      .. code:: console
+      
+         # git clone -b 4.10.2 https://github.com/wazuh/wazuh-dashboard.git
+         # cd wazuh-dashboard/dev-tools/build-packages/base-packages-to-base
 
-   Example:
-   
-   .. code:: console
-   
-      # docker build \
-      --build-arg NODE_VERSION=18.19.0 \
-      --build-arg WAZUH_DASHBOARDS_BRANCH=4.9.0 \
-      --build-arg WAZUH_DASHBOARDS_PLUGINS=4.9.0 \
-      --build-arg WAZUH_SECURITY_DASHBOARDS_PLUGIN_BRANCH=4.9.0 \
-      --build-arg OPENSEARCH_DASHBOARDS_VERSION=2.13.0 \
-      -t wzd:4.9.0 \
-      -f wazuh-dashboard.Dockerfile .
-
-#. Run the Docker image:
+   #. Run the script ``run-docker-compose.sh`` with the following parameters:
 
-   .. code:: console
-   
-      # docker run -d --rm --name wazuh-dashboard-package <TAG_OF_IMAGE> tail -f /dev/null
-   
-   Example:
-   
-   .. code:: console
-   
-      # docker run -d --rm --name wazuh-dashboard-package wzd:4.9.0 tail -f /dev/null
+      -  ``--node-version``: Node version to use in the ``.nvmrc`` file.
+      -  ``-b``, ``--base``: Branch of the Wazuh dashboards repository.
+      -  ``-a``, ``--app``: Branch of the Wazuh dashboards Plugins repository.
+      -  ``-s``, ``--security```: Branch of the Wazuh Security Dashboards Plugin repository.
+      -  ``--arm``: [Optional] Build for arm64 instead of x64.
```

Comment:
```suggestion
      -  ``--node-version``: Specifies the Node version for the ``.nvmrc`` file.
      -  ``-b``, ``--base``: Branch of the Wazuh dashboards repository.
      -  ``-a``, ``--app``: Branch of the Wazuh dashboards Plugins repository.
      -  ``-s``, ``--security```: Branch of the Wazuh Security Dashboards Plugin repository.
      -  ``--arm``: [Optional] Build for arm64 architecture instead of x64.
```

Reply:
Done https://github.com/wazuh/wazuh-documentation/pull/7961/commits/4bf50dd4c253ec17a96007a99a3fcb53b7091fba